assets/queries/k8s/kubernetes_docker_socket_volume/query.rego

```diff
@@ -0,0 +1,55 @@
+package Cx
+
+CxPolicy [ result ] {
+	  document := input.document[i]
+    document.kind == "Pod"
+    metadata:= document.metadata
+    spec := document.spec
+    volumes := spec.volumes
+    volumes[c].hostPath.path == "/var/run/docker.sock"
+
+	
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	   sprintf("metadata.name=%s.spec.volumes.name=%s.hostPath.path", [metadata.name, volumes[c].name]),
+                "issueType":		"IncorrectValue",
+                "keyExpectedValue": sprintf("spec[%s].volumes[%s].hostPath.path is not '/var/run/docker.sock'", [metadata.name, volumes[c].name]),
+                "keyActualValue": 	sprintf("spec[%s].volumes[%s].hostPath.path is '/var/run/docker.sock'", [metadata.name, volumes[c].name])
+              }
+}
+
+CxPolicy [ result ] {
+	  document := input.document[i]
+    document.kind != "Pod"
+    document.kind != "CronJob"
+
+	
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    document.kind,
+                "issueType":		"IncorrectValue",
+                "keyExpectedValue": "document.kind is not Pod or CronJob",
+                "keyActualValue": 	"document.kind is a Pod or CronJob",
+              }
+}
```

Comment:
When the resource is neither a Pod nor a CronJob, it's still evaluated, the path where you find the volume hostPath is different, but it's also supposed to check in Resource.Spec.Template.Spec.

Reply:
@pedro-mimoso Please take a look here, when it's not Pod or CronJob, you're still supposed to check, just in a different path.